dashboard/src/index.tsx

```diff
@@ -1,15 +1,16 @@
 import * as React from "react";
 import * as ReactDOM from "react-dom";
 import * as Modal from "react-modal";
-import { createAxiosInterceptors } from "shared/AxiosInstance";
-import { axios } from "./shared/Auth";
+import * as Axios from "shared/AxiosInstance";
```

Comment:
why are we importing everything?

Reply:
because there is an `axios` element in both `AxiosInstance` and `Auth` I cannot do:

```
import {
  axios,
  createAxiosInterceptors,
  createAxiosInterceptorsWithAuth,
} from "shared/AxiosInstance";
import {axios} from "./shared/Auth";
```

because `axios` is duplicated

Reply:
In any case, I have moved the axios intance from the Auth library to the AxiosInstance. Hopefully that's a bit more readable.

Reply:
(haven't yet looked at the updated code yet) for future reference you can do `import { axios as authAxios } from "./shared/Auth"` (see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import)